app_legacy/Helpers/database/game.php

```diff
@@ -266,7 +267,8 @@ function getGamesListByDev(
                            SUM(CASE WHEN ach.Author = :myPtsDev THEN ach.Points ELSE 0 END) AS MyPoints,
                            SUM(CASE WHEN ach.Author = :myRRDev THEN ach.TrueRatio ELSE 0 END) AS MyTrueRatio,
                            SUM(CASE WHEN ach.Author != :notMyAchDev THEN 1 ELSE 0 END) AS NotMyAchievements,
-                           lbdi.MyLBs,";
+                           lbdi.MyLBs,
+                           ticks.MyOpenTickets,";
```

Comment:
This clause only exists if `$ticketsFlag`. You can see the problem by trying to open the individual dev stats page.